projects/python/simulation/agricultural-simulation-dataset-generator/agricultural-environment-robotti-mavicpro/controllers/generate_robotti_obstacles_datasets/generate_robotti_obstacles_datasets.py

```diff
@@ -0,0 +1,472 @@
+# Copyright 2020-2023 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from controller import Supervisor, Camera, Lidar, GPS
+
+useRobotti = True
+
+if (useRobotti):
+
+    import math
+    import random
+    import pathlib
+    import os
+    import json
+    from datetime import date
+
+    import time
+
+    obj = time.gmtime(0)
+    epoch = time.asctime(obj)
+    # print("The epoch is:",epoch)
+
+    FIELD_SIZE = [40, 14]
```

Comment:
Might be a good idea to move these parameter to a separate file to have them gathered or have a standardized way to load/change them.

Reply:
Okay, will collect the parameters in one header file.

Reply:
@passalis Would you suggest a particular method for collection of parameters in a separate file?